lib/Zyberspace/Telegram/Cli/RawClient.php

```diff
@@ -62,7 +62,12 @@ public function exec($command)
             if (substr($answer, 0, 7) === 'ANSWER ') {
                 $bytes = (int) substr($answer, 7);
                 if ($bytes > 0) {
-                    $string = trim(fread($this->_fp, $bytes + 1));
+                    
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Whitespace found at end of line](https://www.codacy.com/app/zyberspace/php-telegram-cli-client/file/699691092/issues/source?bid=2302274&fileBranchId=2378797#l65)